### PR TITLE
TEST: Linter for tl/mlx5

### PR DESCRIPTION
**Pull request by samnordmann**
## What
set up Linter for TL/MLX5

### CODE REVIEW

**Review by bureddy — APPROVED** [review]

File: .github/workflows/clang-tidy-nvidia.yaml [hunk 1]
```diff
@@ -18,11 +18,16 @@ jobs:
...
         sudo apt-get update
         sudo apt-get install -y --no-install-recommends wget lsb-core software-properties-common
-        sudo apt-get install -y --no-install-recommends libibverbs-dev rdma-core rdmacm-utils libibumad-dev
         wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key| sudo apt-key add -
         sudo apt-add-repository "deb http://apt.llvm.org/focal/ llvm-toolchain-focal-${CLANG_VER} main"
         sudo apt-get install -y --no-install-recommends doxygen doxygen-latex clang-tidy-${CLANG_VER} bear
-    - name: Install extra dependencies
+    - name: Install extra rdma dependencies
+      run: |
+        wget --no-verbose http://content.mellanox.com/ofed/MLNX_OFED-5.9-0.5.6.0/MLNX_OFED_LINUX-5.9-0.5.6.0-ubuntu20.04-x86_64.tgz
```
Comment (Sergei-Lebedev) [inline .github/workflows/clang-tidy-nvidia.yaml:26]: pls make it parameter, e.g. MLNX_OFED_VER=MLNX_OFED-5.9-0.5.6.0

**Review by Sergei-Lebedev — APPROVED** [review]

**Comment by Sergei-Lebedev** [comment]
bot:retest

**Comment by samnordmann** [comment]
bot:retest

**Comment by Sergei-Lebedev** [comment]
bot:retest

**Pull request merged**